// error handling like a pro - try catch
// function divide(a,b){
//     try{
//         if(b==0){
//             throw Error("0 se Divide kiya budbak");
//         }
//         console.log(a/b);
//     }
//     catch(err){
//         console.error(err);
//     }
// }
// divide(12,0);
/////////////////////////////////////////////////////////

